.github/workflows/auto-assign.yml

```diff
@@ -0,0 +1,15 @@
+name: "Auto Assign PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
```

Comment:
Maybe on reopen too? Anything else?

Reply:
> Maybe `edited` too? If a second person starts pushing to the PR then we can make them both share ownership. 

From the [documentation](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=edited#pull_request), `edited` means
```
The title or body of a pull request was edited, or the base branch of a pull request was changed.
```

whereas `synchronize` means
```
A pull request's head branch was updated. For example, the head branch was updated from the base branch or new commits were pushed to the head branch.
```
So we might want `synchronize` over `edited`. 

> The one caveat is that it might get messy with people merging upstream, but if there isn't an easy way to avoid that I think adding them is better than not.

Since the commit message is always something like `Merge {latest branch} into {current branch}`, we might be able to not run the job when we this message (I'm not sure how to do this though.)

Reply:
Works for me.